mongo/collection.go

```diff
@@ -125,7 +125,7 @@ func (coll *Collection) copy() *Collection {
 	}
 }
 
-// Clone creates a copy of this collection with updated options, if any are given.
+// Clone creates a copy of the Collection configured with the given CollectionOptions.
```

Comment:
Specify the effect of any existing options on this collection.   Are they merged with the given options, or are they ignored?

Reply:
done.

Reply:
Still makes it sound like the existing options win, since merge doesn't imply anything about ordering. Perhaps

"The specified options are merged with the exiting options on the Collection, which the specified options taking precedence.

Reply:
I'd be fine with an example.  I think it's a common enough scenario to want to change an option on an existing Collection.